n3fit/src/n3fit/layers/msr_normalization.py

```diff
@@ -51,30 +83,35 @@ def call(self, pdf_integrated, photon_integral):
 
         Parameters
         ----------
-        pdf_integrated: (Tensor(1,None,14))
+        pdf_integrated: (Tensor(1, 14))
             the integrated PDF
-        photon_integral: (Tensor(1)):
-            the integrated photon, not included in PDF
+        photon_integral: (Tensor(1, 1))
+            the integrated photon PDF
 
         Returns
         -------
         normalization_factor: Tensor(14)
             The normalization factors per flavour.
         """
-        y = op.flatten(pdf_integrated)
+        y = pdf_integrated[0]  # get rid of the batch dimension
+        photon_integral = photon_integral[0]  # get rid of the batch dimension
```

Comment:
Does the photon integral need a batch dimension? You could as well remove the `[[` in below (or is it because of symmetry arguments?)

(you can also do directly here photon_integral[0][0] btw)

Reply:
I suppose this was more relevant when I actually joined them together before being passed to this layer, but still having the same shapes makes it easier to change things I think, even if it's a bit superfluous now.